core/test/common/FixedPoint.js

```diff
@@ -147,6 +156,27 @@ contract("FixedPoint", function(accounts) {
     assert(await didContractThrow(fixedPoint.wrapMul(uint_max.sub(web3.utils.toBN("1")), web3.utils.toWei("2"))));
   });
 
+  it("Multiplication, with ceil", async function() {
+    const fixedPoint = await FixedPointTest.new();
```

Comment:
As discussed IRL, a test case I think would be interesting: 1e-18*1e-18 should equal 1e-18 when ceiled.
```
await fixedPoint.wrapMulCeil("1", "1") -> "1"
```

Reply:
done